osmoutils/ibc.go

```diff
@@ -3,11 +3,25 @@ package osmoutils
 import (
 	"encoding/json"
 
```

Comment:
Seeing some diff in `osmoutils` from `main`. Some of the are `accum` package which makes sense but there are also a few other helper changes.

They seem unrelated and acceptable to me but commenting for visibility

Reply:
The diff is as follows:

* The entire accum folder (I dont think we need this, right?)
* module_account.go
  * Just lints #3486
* slice_helper.go
  * Adds containsDuplicate func #2892 (this is for Valset Pref so we dont need this)   
* parsers_test.go
  * Random lints from #3856  
* query_cmd_wrap.go
  * Lints from #3486

Fairly confident all these are not needed

Reply:
Makes sense to me to not have all these, thanks for summarizing